### DISCUSSION TITLE
firefly importer uses wrong "Opposing account (name)" when importing from nordigen

### DISCUSSION THREAD

**Discussion by M1scer** [firefly-iii/firefly-iii#7857] [Firefly III] [answered]
Hi,
i have a simmilar issue than descripten in [this ](https://github.com/orgs/firefly-iii/discussions/7386)Q&A.

In my case i am using the importer to import the transactions via nordigen so i am not able to use the mapping options to ignore the IBAN.

Of course, I can try to create an account for each expense and move it properly via rules. But of course this is not very intuitive and I have to discover all wrongly assigned entries first, so I have to check each new entry carefully.

Is there also the possibility for the import with Nordigen to irgnore the IBAN?

I think the problem here is, that they are using the same payment service provider and therefore of course also the same IBAN

**Comment by JC5** [comment, marked as answer]
This can't really be helped I'm afraid. I keep around one or two generic accounts to catch these payment providers, and then I edit them further.

**Reply by M1scer** [reply]
I could do the same, but often you don't realize it was a payment processor until a "wrong" entry is added.

Can't grouping by name instead of IBAN help here?

I would prefer to merge accounts by rules than to have to muddle accounts apart again.

**Reply by M1scer** [reply]
Is there a way to display the origin "name" of a transaction?

In my bank's online banking the name is displayed correctly, in Firefly this name is replaced by the account name.
So it is no longer visible for me what kind of transaction this actually was.

I have the problem, if I pay in Switzerland with card but I have a German card, all bookings are debited to me from the same IBAN, but the debit name is correct.
Now I have a lot of bookings and have to manually compare all entries with my online banking, because the only helpful information is lost during import.
